ntbs-service/DataMigration/CaseManagerImportService.cs

```diff
@@ -42,46 +42,27 @@ public class CaseManagerImportService : ICaseManagerImportService
             var legacyCaseManager = await _migrationRepository.GetLegacyUserByUsername(legacyCaseManagerUsername);
             var existingCaseManager = await _userRepository.GetUserByUsername(legacyCaseManagerUsername);
             var ntbsCaseManager = existingCaseManager ??
-                                  new User {IsActive = false, Username = legacyCaseManager.Username, CaseManagerTbServices = new List<CaseManagerTbService>()};
+                                  new User {
+                                      IsActive = false,
+                                      IsCaseManager = false,
+                                      Username = legacyCaseManager.Username,
+                                      CaseManagerTbServices = new List<CaseManagerTbService>()
+                                  };
 
             if (!ntbsCaseManager.IsActive)
             {
                 ntbsCaseManager.GivenName = legacyCaseManager.GivenName;
                 ntbsCaseManager.FamilyName = legacyCaseManager.FamilyName;
                 ntbsCaseManager.DisplayName = $"{legacyCaseManager.GivenName} {legacyCaseManager.FamilyName}";
             }
-            ntbsCaseManager.IsCaseManager = true;
-
-            await AddTbServiceToUserBasedOnLegacyPermissions(ntbsCaseManager, legacyTbServiceCode);
+            else
+            {
+                ntbsCaseManager.IsCaseManager = true;
```

Comment:
I don't think we want this logic - I think we want legacy users to default to `IsCaseManager` being `true` so that if they are later added to the TB service they will be a case manager by default.

Or have I missed something?

Reply:
I think you're right, we should always set isCaseManager to true.